BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -385,26 +385,62 @@ public final class ConvivaAnalytics: NSObject {
             value: Int64(player.currentTime(.relativeTime) * 1_000)
         )
     }
-}
 
-private extension Ad {
-    var adInfo: [String: Any] {
+    private func buildAdInfo(adStartedEvent: AdStartedEvent) -> [String: Any] {
         var adInfo = [String: Any]()
+
+        adInfo["c3.ad.id"] = "NA"
```

Comment:
Can we extract the "NA" into a constant for the not-applicable use-case? 🤔

Reply:
Of course. It's NA is not available here though, I think.

https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/71/commits/b9e646ca057a5972f0dd6b5d22919216cfa2c225